<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 153, 102)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 24pt &quot;Dubai Medium&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PILIH MATA PELAJARAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMtk">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>MATEMATIKA</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/box.svg</normaloff>../../../lenovo/Downloads/box.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFsk">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>FISIKA</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/send.svg</normaloff>../../../lenovo/Downloads/send.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIng">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>BAHASA INGGRIS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/type.svg</normaloff>../../../lenovo/Downloads/type.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>KELUAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/log-out.svg</normaloff>../../../lenovo/Downloads/log-out.svg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
